src/chain.rs

```diff
@@ -282,16 +314,23 @@ pub mod kusama {
 		type Solution = NposSolution24;
 
 		// SYNC
-		fn solution_weight(v: u32, _t: u32, a: u32, d: u32) -> Weight {
-			// feasibility weight.
-			(31_722_000 as Weight)
-				// Standard Error: 8_000
-				.saturating_add((1_255_000 as Weight).saturating_mul(v as Weight))
-				// Standard Error: 28_000
-				.saturating_add((8_972_000 as Weight).saturating_mul(a as Weight))
-				// Standard Error: 42_000
-				.saturating_add((966_000 as Weight).saturating_mul(d as Weight))
-				.saturating_add(static_types::DbWeight::get().reads(4 as Weight))
+		fn solution_weight(voters: u32, targets: u32, active_voters: u32, _degree: u32) -> Weight {
+			use pallet_election_provider_multi_phase::{RawSolution, SolutionOrSnapshotSize};
+
+			// Mock a RawSolution to get the correct weight wihout having to do the heavy work.
+			let raw = RawSolution {
+				solution: NposSolution24 {
```

Comment:
here, once you build this fake solution, you should debug assert that: 

```
assert_eq!(solution.voter_count(), a);
assert_eq!(solution.unique_targets(), desired_targets);
```

and then you can be sure that you will get the correct weight.

Reply:
yes sir, good idea

I think we can have `assert_eq` there and not debug assert to catch those in the "burn-in"